rs/p2p/consensus_manager/src/receiver.rs

```diff
@@ -494,21 +491,27 @@ where
                     Ok((artifact, peer_id)) => {
                         let id = artifact.id();
                         // Send artifact to pool
-                        sender.send(UnvalidatedArtifactMutation::Insert((artifact, peer_id)));
+                        if sender.send(UnvalidatedArtifactMutation::Insert((artifact, peer_id))).is_err() {
+                            error!(log, "The receiving side of the channel, owned by the consensus thread, was closed. This should be infallible situation since a cancellation token should be received. If this happens then most likely there is very subnet synchonization bug.");
+                        }
```

Comment:
How come clippy didn't complain about this?

Reply:
@alin-at-dfinity it is in the description, using allow(unused) disabled running clippy for the whole scope :)